docker.Makefile

```diff
@@ -8,6 +8,7 @@ DEV_DOCKER_IMAGE_NAME = docker-cli-dev
 LINTER_IMAGE_NAME = docker-cli-lint
 CROSS_IMAGE_NAME = docker-cli-cross
 MOUNTS = -v `pwd`:/go/src/github.com/docker/cli
+VERSION = $(shell cat VERSION)
```

Comment:
@andrewhsu have you tried without specifying `VERSION` here, now that the LDFLAGS worked? I'm afraid the VERSION now is tied to whatever is in the repo, when in fact it should be tied to the version in the repo that integrates.